.github/workflows/dart.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         - uses: actions/checkout@v2
         - run: pub get
         - name: Test on Chrome
-          run: pub run test -p chrome test/*
+          run: pub run test -p chrome .
```

Comment:
don't we have tests only on https://github.com/getsentry/sentry-dart/tree/main/dart/test ?

Reply:
https://github.com/getsentry/sentry-dart/pull/112#discussion_r504547793 :)

Reply:
I changed it to `pub run test -chrome test`